must-gather/collection-scripts/gather_nro

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+crdoi_list=()
+crdoi_list+=$(oc get crds -o=custom-columns=NAME:.metadata.name --no-headers | grep -E "^*\.nodetopology\.openshift\.io$")
+crdoi_list+=( noderesourcetopologies.topology.node.k8s.io )
+
+# get crds
+for crd_name in ${crdoi_list[@]}
+do
+    # get crd definition
+    oc adm inspect --dest-dir must-gather crd/${crd_name}
+
+    # get crd instances
+    oc adm inspect --dest-dir must-gather ${crd_name}
```

Comment:
what's the benefit of `inspect` over plain `get -o yaml`?

Reply:
`inspect` is the command commonly used by all the `must-gather` scripts. It defaults to a plain `get` but could execute predefined behaviours with some resource types to collect additional resources related with the one asked. 
So in the "worst" case ( not having any behaviour defined ) we get the same we would obtain with `oc get`, on the other hand we could also benefit from any specific behaviour defined.
Also `inspect` uses a defined folder layout to save the collected info to disk if we use `oc get` we should mimic that layout and be aware of any change on it so sync with

Reply:
fair enough, let's keep `inspect` then